src/uct/tcp/tcp_ep.c

```diff
@@ -834,8 +958,8 @@ uct_tcp_ep_am_sendv_get_trace_payload(uct_tcp_am_hdr_t *hdr,
 
     /* If user requested trace data, we copy header and payload
      * to EP TX buffer in order to trace correct data */
-    uct_am_short_fill_data(hdr + 1, *(const uint64_t*)header,
-                           payload_iov->iov_base, payload_iov->iov_len);
+    uct_tcp_ep_am_short_fill_data(hdr, *((const uint64_t*)header),
```

Comment:
what is the reason to change it back?

Reply:
changed by mistake when fixing merge conflicts